core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -324,7 +324,8 @@ class BrokerServer(
                                                     time,
                                                     threadNamePrefix,
                                                     config.metadataSnapshotMaxNewRecordBytes,
-                                                    metadataSnapshotter)
+                                                    metadataSnapshotter
```

Comment:
it would be good to avoid this kind of whitespace change -- it makes merges harder